src/main/java/seedu/mark/ui/ReminderListPanel.java

```diff
@@ -48,17 +59,26 @@ public ReminderListPanel(ObservableList<Reminder> reminders) {
                 reminderList.getChildren().setAll(reminderItems);
             }
         });
-
+        title.setMinWidth(Region.USE_PREF_SIZE);
         reminderList.getChildren().addAll(reminderItems);
+
     }
 
+
+
     public void setReminderListItems() {
         reminderItems.clear();
 
         for (int i = 0; i < reminders.size(); i++) {
-            Label reminder = new Label(reminders.get(i).toString());
+            String index = String.format("%d. ", i + 1);
+            Label reminder = new Label(index + reminders.get(i).toString());
+            reminder.setStyle("-fx-text-fill: white; -fx-padding: 2px;");
```

Comment:
Hm... is it possible to set this in `DarkTheme.css` instead? So that we can easily change anything related to style.

I'm not sure if this works, by you can try defining a style class in the CSS, then setting the style class of each `reminder` Label:
<img width="624" alt="Screenshot 2019-11-08 at 3 36 30 PM" src="https://user-images.githubusercontent.com/46101221/68458457-d7806d00-023d-11ea-8d56-dd79fbe9df92.png">

Reply:
added